portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +25,35 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  public ArrayList<String> randomFacts;
```

Comment:
Can we make randomFacts private? 

Aside from intentionally public APIs, its recommended practice that all remaining classes and members should use the lowest visibility possible.

Reply:
In a later step it actually asks us to remove the stuff that we did in this step, so I don't have a randomFacts variable anymore.